src/import/kv_service.rs

```diff
@@ -289,4 +289,51 @@ impl ImportKv for ImportKVService {
                 .then(move |res| send_rpc_response!(res, sink, label, timer)),
         )
     }
+
+    fn get_version(
+        &mut self,
+        ctx: RpcContext<'_>,
+        _req: GetVersionRequest,
+        sink: UnarySink<GetVersionResponse>,
+    ) {
+        let label = "get_version";
+        let timer = Instant::now_coarse();
+
+        ctx.spawn(
+            self.threads
+                .spawn_fn(|| {
+                    Ok((env!("CARGO_PKG_VERSION"), env!("TIKV_BUILD_GIT_HASH")))
+                })
+                .map(|(v, c)| {
+                    let mut res = GetVersionResponse::new();
+                    res.set_version(v.to_owned());
+                    res.set_commit(c.to_owned());
+                    res
+                })
+                .then(move |res| send_rpc_response!(res, sink, label, timer)),
+        )
+    }
+
+    fn get_metrics(
+        &mut self,
+        ctx: RpcContext<'_>,
+        _req: GetMetricsRequest,
+        sink: UnarySink<GetMetricsResponse>,
+    ) {
+        let label = "get_metrics";
+        let timer = Instant::now_coarse();
+
+        ctx.spawn(
+            self.threads
+                .spawn_fn(|| {
+                    Ok(metrics::dump())
```

Comment:
```suggestion
                    Ok(dump())
```

Reply:
and remove `self` in `use super::metrics::{self, *};`

Reply:
In Rust we tend to avoid `use xyz::*`, and `metrics::dump()` is fine.